src/Company/Models/Company.php

```diff
@@ -0,0 +1,90 @@
+<?php declare(strict_types=1);
+
+namespace VSV\GVQ_API\Company\Models;
+
+use Ramsey\Uuid\UuidInterface;
+use VSV\GVQ_API\Common\ValueObjects\NotEmptyString;
+
+class Company
+{
+    /**
+     * @var UuidInterface
+     */
+    private $id;
+
+    /**
+     * @var NotEmptyString
+     */
+    private $name;
+
+    /**
+     * @var TranslatedAliases
+     */
+    private $aliases;
+
+    /**
+     * @param UuidInterface $id
+     * @param NotEmptyString $name
+     * @param TranslatedAliases $aliases
+     */
+    public function __construct(UuidInterface $id, NotEmptyString $name, TranslatedAliases $aliases)
+    {
+        if ($aliases->count() !== 2 || !$this->hasValidValues($aliases)) {
+            $suppliedValues = '';
+            foreach ($aliases as $alias) {
+                $suppliedValues .= $alias->getAlias()->toNative().' - '.$alias->getLanguage()->toNative().', ';
```

Comment:
Je kan deze strings in een array steken en dan `implode` gebruiken. Dan vermijdt je de extra , voor de het laatste item.